docs/conf.py

```diff
@@ -23,6 +23,10 @@
 settings.LOGGING = {}
 init_hq_python_path()
 
+# Must setup django app registry prior to building docs with sphinx
+import django
```

Comment:
I like this here where it is (my vote is to keep it here).

```suggestion
import django  # noqa: E402
```